packages/contracts-communication/contracts/events/InterchainDBEvents.sol

```diff
@@ -2,36 +2,26 @@
 pragma solidity ^0.8.0;
 
 abstract contract InterchainDBEvents {
-    /// @notice Emitted when a local entry is written to the database.
-    /// @param dbNonce      The nonce of the batch containing the entry.
-    /// @param entryIndex   The index of the entry within the batch.
+    /// @notice Emitted when a local entry (a specific digest by the specific writer) is written to the database.
```

Comment:
> local entry (a specific digest by the specific writer) 

should be simplified

Reply:
https://github.com/synapsecns/sanguine/pull/2599/files#r1595405516

Reply:
Fixed in 2cb17d7b9544a80d8cc9700df20f8b2264671f17